hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/context/properties/SerializableSemVers.java

```diff
@@ -165,6 +165,11 @@ public int compareTo(@Nullable final SoftwareVersion other) {
         }
     }
 
+    @Override
+    public SoftwareVersion buildForDeserialization() {
```

Comment:
Note: nullity annotations are not available in this module without gradle changes